### DISCUSSION TITLE
Not 0 when mo power  !

### DISCUSSION THREAD

**Discussion by vincegre** [JGAguado/Smart_Powermeter#37] [Q&A] [answered]
Hi

Just bought the complete kit and I'm quite impressed by quality of the board and design but I have an annoying issue. Even without CT connected or CT connected but laying on table, ESPHome reports some small consumptions on each input ! I have around 100W measured while it should be 0.
I'm using your default code as you supply. How to fix the problem ?

Thanks

Vincèn

**Comment by JGAguado** [comment]
Hi Vincèn,

Thanks for supporting the project! It can be that you may need to play around the calibration coefficients associated to each CT clamp. @lucazeta1975 opened some days ago a nice [discussion ](https://github.com/JGAguado/Smart_Powermeter/discussions/35) with some updated calibration factors, I'd recommend them to give them a try or check out the [calibration procedure from the docs ](https://smart-powermeter.readthedocs.io/en/v2r2/installation.html#calibration) 😉😉.

Let me know if this helps.

PS: I'm converting this issue to a discussion as it fits better.

**Comment by vincegre** [comment]
Hi @JGAguado 

Thanks a lot for your answer and for moving it as a discussion ;)

For calibrations I have done it once installed in electrical panel and discussion of @lucazeta1975 confirms what I have done too.

My only issue now is that once packed in electrical panel to be able to close it I have some small consmuptions measured that should not be there. Is it i a problem if CT are side by side or very near of each other ?

Best

Vincèn

**Comment by lucazeta1975** [comment, marked as answer]
I share my observations:
[https://github.com/JGAguado/Smart_Powermeter/discussions/35#discussioncomment-10825783](https://github.com/JGAguado/Smart_Powermeter/discussions/35#discussioncomment-10825783)
